spin/apps/07-json-api-llm/magic-8-ball-ts/spin.toml

```diff
@@ -13,4 +13,4 @@ exclude_files = ["**/node_modules"]
 [component.trigger]
 route = "/magic-8"
 [component.build]
-command = "npm run build"
+command = "npm install && npm run build"
```

Comment:
This is fine for a workshop, but it gets really annoying to run install on every build (or on file change..)